engine/src/main/java/org/terasology/rendering/dag/nodes/DownSamplerNode.java

```diff
@@ -58,28 +57,24 @@ public void initialise() {
     /**
      * Initializes the DownSamplerNode instance. This method is meant to be called once, shortly after instantiation.
      *
-     * @param inputConfig an FBOConfig instance describing the input FBO, to be retrieved from the FBO manager
-     * @param inputManager the FBO manager from which to retrieve the input FBO
-     * @param outputConfig an FBOConfig instance describing the output FBO, to be retrieved from the FBO manager
-     * @param outputManager the FBO manager from which to retrieve the output FBO
+     * @param inputFboConfig an FBOConfig instance describing the input FBO, to be retrieved from the FBO manager
+     * @param inputFboManager the FBO manager from which to retrieve the input FBO
+     * @param outputFboConfig an FBOConfig instance describing the output FBO, to be retrieved from the FBO manager
+     * @param outputFboManager the FBO manager from which to retrieve the output FBO
      * @param aLabel a String to label the instance's entry in output generated by the PerformanceMonitor
      */
-    public void initialise(FBOConfig inputConfig, BaseFBOsManager inputManager,
-                           FBOConfig outputConfig, BaseFBOsManager outputManager,
+    public void initialise(FBOConfig inputFboConfig, BaseFBOsManager inputFboManager,
+                           FBOConfig outputFboConfig, BaseFBOsManager outputFboManager,
                            String aLabel) {
+        this.outputFboManager = outputFboManager;
+        this.outputFboUrn = outputFboConfig.getName();
 
-        this.outputFBOconfig = outputConfig;
-        this.outputFBOmanager = outputManager;
-
-        requiresFBO(inputConfig, inputManager);
-        requiresFBO(outputConfig, outputManager);
-
-        outputFBO = outputManager.get(this.outputFBOconfig.getName());
-        outputManager.subscribe(this);
+        requiresFBO(inputFboConfig, inputFboManager);
+        outputFbo = requiresFBO(outputFboConfig, outputFboManager);
```

Comment:
Sorry, I didn't explain myself clearly enough. I'm thinking the initialization pattern:

`fbo = requiresFBO(config, manager);`

instead of using update(), could be more widespread. For example, have a look in DeferredPointLightsNode. I think it could be applicable there, to initialize the readOnlyGBuffer variable. And the same could be done in other nodes, skipping the update() call within initialize.